daemon/mgr/container.go

```diff
@@ -132,8 +132,36 @@ func (mgr *ContainerManager) Restore(ctx context.Context) error {
 }
 
 // Remove removes a container, it may be running or stopped and so on.
-// TODO
 func (mgr *ContainerManager) Remove(ctx context.Context, name string, option *ContainerRemoveOption) error {
+	c, err := mgr.container(name)
```

Comment:
In the future, I think we need classify a StatusNotFound code here, since we may not find container by input name.
We need to do this, in follow-up pull requests.